bin/runtests

```diff
@@ -72,16 +72,18 @@ else
         echo "No cached dump found matching MD5 🐢 🐢 🐢"
     fi
     echo "🔨🔨 Install official/OCA modules 🔨🔨"
-    odoo --stop-after-init --workers=0 --database $DB_NAME_TEST --log-level=warn --without-demo="" -i ${DEPS_ADDONS}
+    gosu odoo odoo --stop-after-init --workers=0 --database $DB_NAME_TEST --log-level=warn --without-demo="" -i ${DEPS_ADDONS}
     if [ "$CREATE_DB_CACHE" == "true" -a ! -z "$CACHED_DUMP" ]; then
         echo "Generate dump $CACHED_DUMP into cache 🐘⮕ 📦"
         mkdir -p "$CACHE_DIR"
         pg_dump -Fp -d $DB_NAME_TEST -O -f "$CACHED_DUMP"
     fi
 fi
 echo "🔧🔧 Install local-src modules 🔧🔧"
-odoo --stop-after-init --workers=0 --database $DB_NAME_TEST --log-level=warn --without-demo="" -i ${LOCAL_ADDONS}
-coverage run --source="${LOCAL_SRC_DIR}" "${ODOO_BIN_PATH}" --stop-after-init --workers=0 --database $DB_NAME_TEST --test-enable --log-level=test --log-handler=":INFO" -u ${LOCAL_ADDONS}
+gosu odoo odoo --stop-after-init --workers=0 --database $DB_NAME_TEST --log-level=warn --without-demo="" -i ${LOCAL_ADDONS}
+
+export COVERAGE_FILE=/home/odoo/.coverage
```

Comment:
I guess this is a required change in order to run the tests as odoo (won't be able to write in `/.coverage`), can you document it in the changelog (if we have volumes using the path we have to adapt it)?